Gemfile

```diff
@@ -11,6 +11,7 @@ gem 'aws-sdk-pinpoint'
 gem 'aws-sdk-pinpointsmsvoice'
 gem 'aws-sdk-ses', '~> 1.6'
 gem 'aws-sdk-sns'
+gem 'aws-sdk-cloudwatchlogs'
```

Comment:
We might still be able to leave this `require: false` since it's explicitly loaded here?

https://github.com/18F/identity-idp/blob/60d38425ab4bc7a1ce1024c03d48d5761c16aa18/lib/reporting/cloudwatch_client.rb#L1

Reply:
good call, done